cabot/rest_urls.py

```diff
@@ -44,16 +44,9 @@ class NoCreateViewSet(mixins.RetrieveModelMixin,
     else:
         viewset_class = viewsets.ModelViewSet
 
-    # Construct the queryset (ie. the list of model objects)
-    arg_queryset = None
-    if issubclass(arg_model, PolymorphicModel):
-        arg_queryset = arg_model.objects.instance_of(arg_model)
-    else:
-        arg_queryset = arg_model.objects.all()
```

Comment:
related to the polymorphicmodel changes?

Reply:
Yup, due to an update to django-polymorphic. Found this change in the main Cabot repo.